### PR TITLE
sort: Move help strings to markdown file

### PR DESCRIPTION
**Pull request by iambasanta**
Move help strings for `sort` to its own markdown file.

### CODE REVIEW

File: src/uu/sort/sort.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+```
+sort [OPTION]... [FILE]...
+```
+
+Display sorted concatenation of all FILE(s). With no FILE, or when FILE is -, read standard input.
+
+## After help
+
+The key format is FIELD[.CHAR][OPTIONS][,FIELD[.CHAR]][OPTIONS].
```
Comment (sylvestre) [inline src/uu/sort/sort.md:11]: please ignore this warning

Comment (sylvestre) [inline src/uu/sort/sort.md:11]:
```suggestion
The key format is `FIELD[.CHAR][OPTIONS][,FIELD[.CHAR]][OPTIONS]`.
```

File: src/uu/sort/sort.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Fields by default are separated by the first whitespace after a non-whitespace character. Use -t to specify a custom separator.
+In the default case, whitespace is appended at the beginning of each field. Custom separators however are not included in fields.
+
+FIELD and CHAR both start at 1 (i.e. they are 1-indexed). If there is no end specified after a comma, the end will be the end of the line.
+If CHAR is set 0, it means the end of the field. CHAR defaults to 1 for the start position and to 0 for the end position.
+
+Valid options are: MbdfhnRrV. They override the global options for this key.
```
Comment (sylvestre) [inline src/uu/sort/sort.md:19]: please add this to the ignore spell

Comment (sylvestre) [inline src/uu/sort/sort.md:19]:
```suggestion
Fields by default are separated by the first whitespace after a non-whitespace character. Use `-t` to specify a custom separator.
In the default case, whitespace is appended at the beginning of each field. Custom separators however are not included in fields.

`FIELD` and `CHAR` both start at 1 (i.e. they are 1-indexed). If there is no end specified after a comma, the end will be the end of the line.
If `CHAR` is set 0, it means the end of the field. `CHAR` defaults to 1 for the start position and to 0 for the end position.

Valid options are: `MbdfhnRrV`. They override the global options for this key.
```

Comment (iambasanta) [reply src/uu/sort/sort.md:19]: okey

File: src/uu/sort/sort.md [hunk 3]
```diff
@@ -8,12 +8,12 @@ Display sorted concatenation of all FILE(s). With no FILE, or when FILE is -, re
...
+Fields by default are separated by the first whitespace after a non-whitespace character. Use `-t` to specify a custom separator.
 In the default case, whitespace is appended at the beginning of each field. Custom separators however are not included in fields.
 
-FIELD and CHAR both start at 1 (i.e. they are 1-indexed). If there is no end specified after a comma, the end will be the end of the line.
-If CHAR is set 0, it means the end of the field. CHAR defaults to 1 for the start position and to 0 for the end position.
+`FIELD` and `CHAR` both start at 1 (i.e. they are 1-indexed). If there is no end specified after a comma, the end will be the end of the line.
+If `CHAR` is set 0, it means the end of the field. `CHAR` defaults to 1 for the start position and to 0 for the end position.
 
-Valid options are: MbdfhnRrV. They override the global options for this key.
+Valid options are: `MbdfhnRrV`. They override the global options for this key.
```
Comment (sylvestre) [inline src/uu/sort/sort.md:19]: add

to the top of the file

**Pull request merged**